setup.py

```diff
@@ -17,6 +17,19 @@
 with open(os.path.join(here, 'sewer', '__version__.py'), 'r') as f:
     exec(f.read(), about)
 
+dns_provider_deps_map = {
+    'aliyun': ['aliyun-python-sdk-core-v3', 'aliyun-python-sdk-alidns'],
+    'alicloud': ['aliyun-python-sdk-core-v3', 'aliyun-python-sdk-alidns'],
+    'he': ["hurricanedns"],
+    'hedns': ["hurricanedns"],
```

Comment:
@menduo can we just use either `he` or `hedns` rather than both of them? 
since we already have `cloudflare`, `aurora` I think we should just call it `hurricane`

Reply:
OK, I agree.